images/installer/nested-libvirt/README.md

```diff
@@ -0,0 +1,40 @@
+## Image to enable launching a libvirt cluster nested in a GCE instance, for libvirt CI testing.
```

Comment:
nit: The initial title for a Markdown file should use an h1 `#`, with subsequent subsections incrementing by one level.

Also, this is a long title ;).  How about `# Nested libvirt on GCE`, with the other points covered in the paragraph(s) after the title?

Reply:
thanks, fixed. :)